lib/config/flat-config-helpers.js

```diff
@@ -65,13 +65,9 @@ function parseRuleId(ruleId) {
  *      or undefined if not.
  */
 function getRuleFromConfig(ruleId, config) {
-
     const { pluginName, ruleName } = parseRuleId(ruleId);
 
-    const plugin = config.plugins && config.plugins[pluginName];
-    const rule = plugin && plugin.rules && plugin.rules[ruleName];
-
-    return rule;
+    return config?.plugins?.[pluginName]?.rules?.[ruleName];
```

Comment:
Thanks for the PR @lucasrmendonca. I seems that the `config` argument is not optional in this function. So maybe like this?

```suggestion
    return config.plugins?.[pluginName]?.rules?.[ruleName];
```

Reply:
Done!